core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -132,6 +133,16 @@ oppia.directive('explorationSummaryTile', [
               if ($scope.getCollectionId()) {
                 result += ('?collection_id=' + $scope.getCollectionId());
               }
+              if ($scope.parameters) {
+                var parameterList = $scope.parameters.split('&');
```

Comment:
Two things:
- $scope.parameters should contain a list of parent exp IDs, not an unparsed string.
- We ought to have a standard way to add these attrs to a URL. Could you add a method in urlService for that? It should take in a base URL, and a dict of query parameter key-value pairs. It should then encode those accordingly (if needed) and tack them on to the URL. There might be standard Angular methods for this. Please add Karma tests for that, too.

Reply:
Ok, will try that.

Reply:
Done, I used $location.search() for this purpose.